shell/common/native_mate_converters/file_dialog_converter.cc

```diff
@@ -8,6 +8,7 @@
 #include "shell/browser/api/atom_api_browser_window.h"
```

Comment:
Api header shouldn't be needed now.

Reply:
We need it in [L.65](https://github.com/electron/electron/pull/19577/files#diff-2c4b6a568b2c88fa116f70644e3fd219R65), at least my compiler says so 😄.